doc/sign-package.md

```diff
@@ -104,6 +104,7 @@ Settings | Type | Description
 ~~UserMemSize~~ | `int \| string` | Deprecated, use `MemorySize` instead
 MemorySize | `int \| string` | Amount of memory your application needs to run. Try not to make this just a very large number as the larger this number needs to be the slower load time will be. Value can be bytes (just a number), **k**ilobytes (for example `"128k"`), **m**egabytes (for example `"512m"`), or **g**igabytes (for example `"1g"`)
 MainStackSize | `int \| string` | Stack size of your application's main process. Defaults to 1536k (or 1.5M) bytes. Normally, you do not need to customize this. If running an application generates a OOM error like in [#612](https://github.com/deislabs/mystikos/issues/612), try tuning this value, e.g. to 8M. Value can be bytes (just a number), **k**ilobytes (for example `"128k"`), **m**egabytes (for example `"512m"`), or **g**igabytes (for example `"1g"`)
+ThreadStackSize | `int \| string` | The default stack size of pthreads created by the application.
```

Comment:
The description does not seem to be accurate; the size set by this configuration will only be applied if it's greater than a default value (defined in musl?)